README.md

```diff
@@ -31,11 +31,15 @@ below.
 2) Executing the program.  
   a. `python3 main.py`: Start the program's CLI menu.  
   b. `python3 main.py --help`: See the instructions for use as a command.  
-3) `exit`: Close the virtual environment.
+3) `exit`: Close the virtual environment
 
```

Comment:
This is very nitpicky but can you add the period back in? This document is externally facing so I feel it would be valuable to be consistent with the other options right there, as well as good grammar. It probably just got deleted by accident.

Reply:
Huh, kinda weird... I have the period on mine but for some reason the README I am pushing does not have that.